operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -874,4 +874,6 @@ spec:
   minKubeVersion: 1.15.0
   provider:
     name: Red Hat
+  skips:
+  - rhacs-operator.v4.1.0
```

Comment:
Does this yaml file preserve yaml comments? If yes, I would strongly recommend adding a comment giving a reference to why 4.1.0 is listed as broken.

Reply:
@msugakov Seems like it doesn't preserve comments.